examples/pytorch/densecl.py

```diff
@@ -0,0 +1,117 @@
+# Note: The model and training settings do not follow the reference settings
+# from the paper. The settings are chosen such that the example can easily be
+# run on a small dataset with a single GPU.
+
+import copy
+
+import torch
+import torchvision
+from torch import nn
+
+from lightly.loss import NTXentLoss
+from lightly.models import utils
+from lightly.models.modules import DenseCLProjectionHead
+from lightly.transforms import DenseCLTransform
+from lightly.utils.scheduler import cosine_schedule
+
+
+class DenseCL(nn.Module):
+    def __init__(self, backbone):
+        super().__init__()
+        self.backbone = backbone
+        self.projection_head_global = DenseCLProjectionHead(512, 512, 128)
+        self.projection_head_local = DenseCLProjectionHead(512, 512, 128)
```

Comment:
From the paper
> In fact, the dense projection head has the
same number of parameters as the global projection head.

I didn't add a 1x1 conv head because it is exactly the same as the head using MLPs.

Reply:
Yes, I know, that is why I said "explicitly use" a 1x1 convolutional layer. I think it would be clearer what the implementation is.

Reply:
I did it that way because we anyways need the features in `(B, N, C)` format for the matching part afterwards. This avoids having to reshape the features in different places.